setup.cfg

```diff
@@ -16,9 +16,14 @@ combine_as_imports = true
 
 [mypy]
 python_version = 3.11
-plugins = mypy_django_plugin.main,mypy_drf_plugin.main
-ignore_missing_imports = True
-allow_untyped_globals = True
+plugins = mypy_django_plugin.main, mypy_drf_plugin.main
+ignore_missing_imports = true
+allow_untyped_globals = true
+# TODO: Enable this, but it requires a lot of work (~600 errors)
+check_untyped_defs = false 
```

Comment:
Didn't know this was a thing, but we have been suppressing roughly 600 errors because of it. Should perhaps enable and set an allowed error amount (like we've done for typescript in the webapp)? Thoughts?

Reply:
Also, we should configure `mypy` in general a lot better here! There's many features which are currently not utilizing.

Reply:
I agree. Setting the limit to what it is now at least enforces all new code to be typed :)
When we set up mypy last time, it was just the bootstrapping, and we ignored all types just to make it pass CI. But I think enabling some of the actual nice features would be nice.